### PR TITLE
Correct dependency on avro

### LINKED ISSUE #6: data_pipeline imports avro but does not list it as a dependency
https://github.com/Yelp/data_pipeline/blob/b0a6609188146b9da7d415384047359ceacfb218/data_pipeline/envelope.py#L22-L23

### PR DESCRIPTION
**Pull request by asottile**
Resolves #6

### CODE REVIEW

**Comment by asottile** [comment]
@abrarsheikh ping 4 weeks

File: setup.py [hunk 1]
```diff
@@ -41,6 +41,7 @@
     packages=find_packages(exclude=['tests*', 'benchmarks*']),
     include_package_data=True,
     install_requires=[
+        'avro',
```
Comment (abrarsheikh) [inline setup.py]: the project does not list avro as dependency but instead uses [yelp-avro==1.9.2](https://pypi.python.org/pypi/yelp_avro/1.9.2), which is yelp's version of avro(this version has extra functionality which are absent in apache/avro).

Comment (asottile) [inline setup.py]: That's false though, it [imports vanilla avro right here](https://github.com/Yelp/data_pipeline/blob/b0a6609188146b9da7d415384047359ceacfb218/data_pipeline/envelope.py#L22-L23)

Comment (abrarsheikh) [inline setup.py]: if you install yelp-avro it actually creates a package with name `avro`. That is how we are able to import just avro. 
I am not completely happy with the way its implemented but couldn't think of any other neat way.
@jian-yelp thoughts?

**Comment by asottile** [comment]
Oh then this is probably broken in yelp main where both are installed

**Comment by asottile** [comment]
Turns out we're just installing this incorrectly in yelp-main

**Pull request closed without merging**